src/managed_data/schema.py

```diff
@@ -35,13 +36,51 @@
     "reference_ids",
 }
 
+
+Cardinality = Literal["one", "many"]
+
 IDFormat = Literal["human-id", "system-id"]
 """Format of an ID"""
 
 DatabaseReturnType = Literal["dataframe", "dict"]
 """Return type of a database"""
 
 
+class CreateDatabaseResponse(BaseModel):
```

Comment:
Do you think this kind of stuff could be generated from an openapi schema in the future?